docs/quickstarts/rbac-customizing-access/rbac-customizing-access.yaml

```diff
@@ -0,0 +1,107 @@
+# Additional info: https://docs.openshift.com/container-platform/4.9/web_console/creating-quick-start-tutorials.html
```

Comment:
Can you please change this file to be `yml` not `yaml`. I know its silly, but we currently only support `yml` extension. I will make sure we support both in a near future.

Reply:
No worries! All fixed :)